Core/Src/dti.c

```diff
@@ -251,6 +280,23 @@ static void dti_record_rpm(dti_t *mc, can_msg_t msg)
 	osMutexRelease(*mc->mutex);
 }
 
+uint16_t dti_get_input_voltage(dti_t *mc)
+{
+	uint16_t voltage;
+	osMutexAcquire(*mc->mutex, osWaitForever);
+	voltage = mc->input_voltage;
+	osMutexRelease(*mc->mutex);
+
+	return voltage;
+}
+
+static void dti_set_input_voltage(dti_t *mc, can_msg_t msg)
```

Comment:
why we need any of this voltage stuff?

Reply:
I wrote this code, don't remember why I did it, and have no clue what I was thinking when I did.